docs/site/index/survey-tool.md

```diff
@@ -16,7 +16,9 @@ Below are the main pages to look at.
 
 ### Schedule
 
-For the Milestone schedule, see the navigation bar on the left.
+For the Milestone schedule, see the [CLDR Schedule](/index/draft-schedule). See the [Information Hub for Linguists](/translation) for the most recent updates about the current or upcoming cycle.
+
+The SurveyTool usually is open for one submission period annually starting in Q2, but schedule may change depending on needs.
```

Comment:
IMHO, you removed a "the" that should be there I think?
The SurveyTool usually is open for one submission period annually starting in Q2, but **the** schedule may change depending on needs.

Reply:
The icon that looks like a page symbol with ± helps to speed things up, BTW. Hitting it creates a comment that can be merged with a button. Like the following.
```suggestion
The SurveyTool usually is open for one submission period annually starting in Q2, but the schedule may change depending on needs.
```

Reply:
It also shows the delta change nicely.

Reply:
> It also shows the delta change nicely.

TIL